property_tests/arbitraries/candid/primitive/bool.ts

```diff
@@ -1,5 +1,19 @@
 import fc from 'fast-check';
-import { CandidMetaArb } from '../candid_arb';
+import { SimpleCandidValueAndMetaArb } from '../simple_type_arbs/value_and_meta_arb';
+import { SimpleCandidDefinitionArb } from '../simple_type_arbs/definition_arb';
+import { SimpleCandidValuesArb } from '../simple_type_arbs/values_arb';
 import { booleanToSrcLiteral } from '../to_src_literal/boolean';
+import { BoolCandidDefinition } from '../definition_arb/types';
+import { CandidValues } from '../values';
 
-export const BoolArb = CandidMetaArb(fc.boolean(), 'bool', booleanToSrcLiteral);
+export const BoolArb = SimpleCandidValueAndMetaArb(
```

Comment:
SimpleCandidValueAndMetaArb should cease to exist and be replaced with ComplexCandidValueAndMetaArb...rename it to CandidValueAndMetaArbCreator or something